torch/csrc/jit/export.cpp

```diff
@@ -568,7 +568,7 @@ RawDataExportMap ToModelProto(
   }
 
   model_proto->set_producer_name("pytorch");
-  model_proto->set_producer_version("0.3");
+  model_proto->set_producer_version("0.4");
```

Comment:
nit: I thought there was a version field somewhere in pytorch produced by CMake. But maybe it's only in python.

Reply:
Confirmed, not available in C++ yet.